recipe/build.sh

```diff
@@ -4,10 +4,15 @@ python ${RECIPE_DIR}/brand_python.py
 
 if [ `uname` == Darwin ]; then
     export CFLAGS="-I$PREFIX/include $CFLAGS"
-    export LDFLAGS="-L$PREFIX/lib -headerpad_max_install_names $LDFLAGS"
+    export LDFLAGS="-Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -headerpad_max_install_names $LDFLAGS"
     sed -i -e "s/@OSX_ARCH@/$ARCH/g" Lib/distutils/unixccompiler.py
-    ./configure --enable-shared --enable-ipv6 --with-ensurepip=no \
-        --prefix=$PREFIX
+    ./configure \
+        --enable-ipv6 \
+        --enable-shared \
+        --prefix=$PREFIX \
+        --with-ensurepip=no \
+        --with-tcltk-includes="-I$PREFIX/include" \
+        --with-tcltk-libs="-L$PREFIX/lib -ltcl8.5 -ltk8.5"
```

Comment:
So, the other reason I was asking for this flag is I have found things that do link to the system `gcc`'s libraries often don't work on 10.11 (maybe the libraries are removed? will need to check when I have 10.11 system to try with). Again I don't see why it should cause issues for us with `python` given it is just C, but it is something to be mindful of.